CSS logical properties and values (#29361)
mdn/content: files/en-us/web/css/css_logical_properties_and_values/index.md

- Inline dimension
  - : The dimension parallel to the flow of text within a line, i.e., the horizontal dimension in horizontal writing modes, and the vertical dimension in vertical writing modes. For standard English text, it is the horizontal dimension.

### New properties and values

> CSS was originally designed with only physical coordinates in its controls. The module defines new flow–relative equivalents for [values](/en-US/docs/Web/CSS/CSS_Values_and_Units) and [properties](/en-US/docs/Glossary/Property/CSS).

Reviewer: ```suggestion
CSS was initially designed with only physical coordinates in its controls. The module defines new flow–relative equivalents for many [values](/en-US/docs/Web/CSS/CSS_Values_and_Units) and [properties](/en-US/docs/Glossary/Property/CSS).
```

> Some physical properties now have logical equivalents, while properties that accepted only physical values (`top`, `bottom`, `left`, `right`) now accept flow-relative logical values (`block-start`, `block-end`, `inline-start`, `inline-end`).

Reviewer: ```suggestion
Some physical properties now have logical equivalents. Properties that accept physical values (`top`, `bottom`, `left`, `right`) now also accept flow-relative logical values (`block-start`, `block-end`, `inline-start`, `inline-end`).
```

> 
> ### Not support properties and values
> 
> The transition to logical axes is not fully defined by the module, some properties don't yet have logical equivalents.

Reviewer: ```suggestion

The transition to logical axes is ongoing and not fully defined by the module; some properties don't yet have logical equivalents.
```

Reviewer: removing the header because we can just add the sentence this section contains to the previous section

- `offset-inline-end` {{Non-standard_Inline}} {{Deprecated_Inline}} (now {{CSSxRef("inset-inline-end")}})
- `offset-inline-start` {{Non-standard_Inline}} {{Deprecated_Inline}} (now {{CSSxRef("inset-inline-start")}})

### Unsupported properties

> The folowing properties doesn't have logical equivalents:

Reviewer: ```suggestion
The following properties don't have logical equivalents:
```

- {{CSSxRef("background-position-x")}}
- {{CSSxRef("background-position-y")}}

### Unsupported values

> The folowing properties accept only physical values:

Reviewer: ```suggestion
The following properties accept only physical values:
```
